setup.py

```diff
@@ -79,6 +79,7 @@
             'htseq = multiqc.modules.htseq:MultiqcModule',
             'jellyfish = multiqc.modules.jellyfish:MultiqcModule',
             'kallisto = multiqc.modules.kallisto:MultiqcModule',
+            'disambiguate = multiqc.modules.disambiguate:MultiqcModule',
```

Comment:
I usually try to keep this list alphabetically sorted (not that it actually matters).